homeassistant/components/media_player/denonavr.py

```diff
@@ -167,7 +169,10 @@ def source_list(self):
     @property
     def supported_media_commands(self):
         """Flag of media commands that are supported."""
-        return SUPPORT_DENON
+        if self._current_source in self._receiver._playing_func_list:
```

Comment:
Good idea. I think you should use **self._receiver.netaudio_func_list** instead of _playing_func_list
playing_func_list sources show what they are currently playing, but only for netaudio_func_list sources the play, pause, next, previous commands will work. 
I just created version 0.3.1 of the library, which include properties for netaudio_func_list and playing_func_list for this. You could use them.